community/kernel/src/main/java/org/neo4j/helpers/collection/Visitable.java

```diff
@@ -19,7 +19,7 @@
  */
 package org.neo4j.helpers.collection;
 
-public interface Visitable<T>
+public interface Visitable<V>
 {
-    void accept(Visitor<T, RuntimeException> visitor);
+    void accept( V visitor );
```

Comment:
The existing abstraction was only used in one place. Changing it to the more general type did not loose any information, the existing use was just changed to Visitable<Visitor<..>>. I don't see the point of having two abstractions that are almost identical and each of them is only used in one place vs. having only one abstraction that at least can be used in two places.